bundles/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoPressureUnit.java

```diff
@@ -52,11 +53,11 @@ public static NetatmoPressureUnit fromString(String pressureUnit) {
 	
 	public BigDecimal convertPressure(BigDecimal value) {
 		if (this == DEFAULT_PRESSURE_UNIT) {
-			return value;
+			return value.setScale(0, RoundingMode.HALF_UP);
 		} else if (this == INHG) {
-			return value.multiply(MBAR_TO_INHG);
+			return value.multiply(MBAR_TO_INHG).setScale(2, RoundingMode.HALF_UP);
```

Comment:
My only concern was the risk of possibly accumulating rounding errors, if for example, persistence functions are used on sets of these values.  You may know this to not be a concern; just pointing it out!